CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Changelog
 
+### v4.0.0.preview19-with-filtering - 2023-03-16
```

Comment:
Question: will you also make these same changes to the VRDR Java library?  Some jurisdictions are using the Java library.

Reply:
I can do, I think the idea, for now, is to get the filtering logic onto the .net VRDR library and get some assistance with our filtering logic.
We had issues previously and decided to remove the filtering from our code to the .net VRDR library to allow for other parties to view this logic easily. Once we have got confirmation that the logic is good, then filtering can be used from this library.
At that point I'll make these changes to the java VRDR library.

Reply:
Okay we will consider this a C# library change only for now, I've created an issue to track the Java change in the future: https://github.com/nightingaleproject/VRDR_javalib/issues/14